mods/d2k/rules/vehicles.yaml

```diff
@@ -442,7 +442,7 @@ deviator:
 		BuildPaletteOrder: 40
 		BuildDuration: 373
 		BuildDurationModifier: 40
-		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry, Aircraft\n \n  Atreides:      +Range\n  Harkonnen:  +Health\n  Ordos:          +Speed
+		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry
```

Comment:
For my taste, the description is too long, which leads to these wide info pop-ups, but I have no idea how to shorten it or how to provide this information at one glance. :/

Tank Description
|:----------:|
![tankdesc](https://cloud.githubusercontent.com/assets/20945684/22426387/55e46884-e6ff-11e6-9ba6-c4efacabf4c0.jpg)

Everything else of the PR is fine.